tools/gulptasks/dist-compress.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
+const fs = require('fs-extra');
+const zlib = require('zlib');
+const glob = require('glob');
+const log = require('./lib/log');
+const build = require('./lib/build');
+const yargs = require('yargs');
+const zip = require('gulp-zip');
+const stream = require('stream');
+const util = require('util');
+
+
+const DIST_DIR = 'build/dist';
+
+
+/**
+ * Creates zip files for highcharts, highstock, highmaps, highcharts-gantt.
+ *
+ * @return {Promise<any> | Promise | Promise} Promise to keep
```

Comment:
Confusing type

Reply:
Changed return type to Promise<*> for now.